tests/test_elements.py

```diff
@@ -1140,9 +1142,181 @@ def test_multipole_tilt_90_deg(test_context):
     pf.move(_context=xo.context_default)
     pfy.move(_context=xo.context_default)
 
-    xo.assert_allclose(pf.x, pfy.x, rtol=0, atol=1e-14)
-    xo.assert_allclose(pf.y, pfy.y, rtol=0, atol=1e-14)
-    xo.assert_allclose(pf.px, pfy.px, rtol=0, atol=1e-14)
-    xo.assert_allclose(pf.py, pfy.py, rtol=0, atol=1e-14)
-    xo.assert_allclose(pf.zeta, pfy.zeta, rtol=0, atol=1e-14)
-    xo.assert_allclose(pf.ptau, pfy.ptau, rtol=0, atol=1e-14)
+    assert np.allclose(pf.x, pfy.x, rtol=0, atol=1e-14)
+    assert np.allclose(pf.y, pfy.y, rtol=0, atol=1e-14)
+    assert np.allclose(pf.px, pfy.px, rtol=0, atol=1e-14)
+    assert np.allclose(pf.py, pfy.py, rtol=0, atol=1e-14)
+    assert np.allclose(pf.zeta, pfy.zeta, rtol=0, atol=1e-14)
+    assert np.allclose(pf.ptau, pfy.ptau, rtol=0, atol=1e-14)
+
+
+@for_all_test_contexts
+def test_ecooler(test_context):
+    """Test the electron cooler by comparing the cooling rate and cooling force with Betacool for LEIR.
+    """
+    data = np.load('../test_data/electron_cooler/emittance_betacool.npz')
```

Comment:
To get the test data folder use the strategy here:
https://github.com/xsuite/xtrack/blob/43e6bddcfbaec9e53ae76d9007fc9cc771adc60c/tests/test_coasting.py#L8

https://github.com/xsuite/xtrack/blob/43e6bddcfbaec9e53ae76d9007fc9cc771adc60c/tests/test_coasting.py#L15